spec/system/claims/schools/claims/change_claim_on_check_page_spec.rb

```diff
@@ -55,11 +55,7 @@
     then_i_see_the_error("Select a mentor")
     when_i_check_the_mentor(mentor2)
     when_i_click("Continue")
-    when_i_click("Continue")
-    then_i_see_the_error("Select the number of hours")
```

Comment:
Why did we remove these specs where we check for errors?

Reply:
The spec reads:

```ruby
then_i_expect_the_mentors_to_be_checked([mentor1, mentor2])
when_i_uncheck_the_mentors([mentor1, mentor2])
when_i_click("Continue")
then_i_see_the_error("Select a mentor")
when_i_check_the_mentor(mentor2)
```

Which means `mentor2`'s hours are already persisted and we move onto the next mentor **without** hours.